src/app/workflow/launch-third-party/launch-third-party.component.ts

```diff
@@ -146,6 +146,22 @@ export class LaunchThirdPartyComponent extends Base implements OnChanges, OnInit
    */
   workflowPathAsQueryValue: string;
 
+  cgcTooltip$: Observable<string> = this.descriptorsQuery.hasContent$.pipe(
```

Comment:
(Combinelatest or zip) + pipe + map seems easier.  What was the benefit to doing it this way (2 maps, 2 pipes, and mergemap?

Reply:
I'm used to using just map and flatmap (mergemap in TS) in Java/Scala, and don't know all the RXJS operators very well... Will update